hack/start-operator.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Why would you run this script instead of make targets? I'm ok with this, but maybe we should use this script on kind workflow to have it all fully tested. ?